src/EnergyPlus/HVACManager.cc

```diff
@@ -442,6 +442,7 @@ namespace HVACManager {
 
         if (UseZoneTimeStepHistory) PreviousTimeStep = TimeStepZone;
         for (SysTimestepLoop = 1; SysTimestepLoop <= NumOfSysTimeSteps; ++SysTimestepLoop) {
+            if (DataGlobals::stopSimulation) break;
```

Comment:
Not sure how fast the break should be, and this is a good spot for the master break, but there is another place in code where the entire HVAC system is simulated in a [while loop](https://github.com/NREL/EnergyPlus/pull/8086/files#diff-cd290070b0d136de4179479f221302dbR910-R913). Some complex systems could take some time to complete if not quickly converged. Just a suggestion.

Reply:
Agreed. I will add a break down in the HVAC manager. See I wasn't concerned about that because my use case is circumventing EnergyPlus's HVAC manager and replacing with Modelica. For completion of the feature I think your idea is correct.

Reply:
I added this https://github.com/NREL/EnergyPlus/pull/8086/commits/52e942a8ef8ee2c79c6821856f3c620ae3cdbecf